### ISSUE TITLE
SQLite `sqlBatch` type checking is wrong

### ISSUE THREAD

**Issue by MichaelFedora** [ionic-team/ionic-native#1596]
**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**

Currently this is what the type-checking checks for:

```typescript
/**
 * @param sqlStatements {Array<string | string[]>}
 * @returns {Promise<any>}
 */
 @CordovaInstance()
 sqlBatch(sqlStatements: Array<string | string[]>): Promise<any> { return; }
```

**Expected behavior:**

However, this is what it should be:

```typescript
/**
 * @param sqlStatements {Array<string | [ string, any[] ]>}
 * @returns {Promise<any>}
 */
 @CordovaInstance()
 sqlBatch(sqlStatements: Array<string | [ string, any[] ]>): Promise<any> { return; }
```

**Steps to reproduce:**

You can check the ***official plugin repo*** source/test script here to verify what I said was correct: https://github.com/litehelpers/Cordova-sqlite-storage/blob/storage-master/spec/www/spec/sql-batch-test.js

i.e.:
```javascript
db.sqlBatch([
  'DROP TABLE IF EXISTS MyTable',
  'CREATE TABLE MyTable (data)',
  [ 'INSERT INTO MyTable VALUES (?)', [101] ],
  [ 'INSERT INTO MyTable VALUES (?)', [-101] ],
  [ 'INSERT INTO MyTable VALUES (?)', [123.456] ],
  [ 'INSERT INTO MyTable VALUES (?)', [-123.456] ],
  [ 'INSERT INTO MyTable VALUES (?)', [1234567890123] ],
  [ 'INSERT INTO MyTable VALUES (?)', [-1234567890123] ],
  [ 'INSERT INTO MyTable VALUES (?)', [0] ],
], function() { /* ... */ }
```

**Other information:**

**package.json info:**

```json
"@ionic-native/sqlite": "3.10.3"
```

**Comment by arsenk01** [comment]
Does this fix included in 3.12.1 version?

**Comment by MichaelFedora** [comment]
This was closed in version 4.0.0 according to the [Changelog](https://github.com/ionic-team/ionic-native/blob/master/CHANGELOG.md).

(technically it was a lazy fix because it was just made into an `any[]` type instead of the correct `Array<string | [ string, any[] ]>`, but it still fixed it failing to compile).
